### ISSUE TITLE
Form labels for the access permissions modal

### ISSUE THREAD

**Issue by awead** [psu-stewardship/scholarsphere-4#478]
We're using a model to edit access permissions and it needs some attention.

## WDLL
* add appropriate a11y label to the search form for finding existing users
* "+" for adding permissions should be labeled

**Comment by awead** [comment]
Made obsolete in #654
